pypolygames/tests/test_mcts.py

```diff
@@ -11,9 +11,10 @@
 from .. import params
 from .. import evaluation
 from .. import utils
+from polygames import init_threads
 
 
-@pytest.mark.parametrize("game_name", [game_name for game_name in utils.listings.games()])
+@pytest.mark.parametrize("game_name", ["TicTacToe"])
```

Comment:
Is there a reason this was changed to TicTacToe specifically? Isn't this testing all the games?

Reply:
Yeah, I changed it to only TicTacToe because the tests were taking a **really** long time to run and I wasn't sure how much benefit all the others provided.

Maybe we should test a few games, but all of them makes the CI way too long. It should be ~5 minutes.